src/components/button/__tests__/Button.test.tsx

```diff
@@ -45,11 +42,7 @@ describe('components/button/Button', () => {
         const preventDefault = sinon.spy();
         const stopPropagation = sinon.spy();
 
-        const wrapper = shallow(<Button isDisabled onClick={onClickHandler} />);
-
-        const contains = sinon.stub();
-        contains.withArgs('is-disabled').returns(true);
-        wrapper.instance().btnElement = { classList: { contains } };
+        const wrapper = shallow<Button>(<Button isDisabled onClick={onClickHandler} />);
```

Comment:
shallow<TYPE> indicates that the wrapper.instance() will return that TYPE and you can call functions or methods directly on that instance. Think of it syntactical sugar over casting.

Reply:
A lot of our tests are not currently type-safe due to missing required props, stubbed objects, etc. Will take some effort to migrate them.

Reply:
indeed